doc/website/getting-started/overview.md

```diff
@@ -183,11 +183,12 @@ The service model of iceoryx is derived from AUTOSAR and is still used in the AP
 
 The following table gives an overview of the different terminologies and the current mapping:
 
-|         | Group   | Instance         | Topic                  |
-|---------|---------|------------------|------------------------|
-| ROS2    | Type    | Namespace::Topic | -                      |
-| AUTOSAR | Service | Instance         | Event                  |
-| DDS     | -       | -                | /Group/Instance/Topic  |
+|                                                                                   | Group   | Instance         | Topic                  |
+|-----------------------------------------------------------------------------------|---------|------------------|------------------------|
+| [rmw_iceoryx](https://github.com/ros2/rmw_iceoryx/)                               | Type    | Namespace::Topic | -                      |
+| AUTOSAR                                                                           | Service | Instance         | Event                  |
+| [DDS Gateway](https://github.com/eclipse-iceoryx/iceoryx/tree/master/iceoryx_dds) | -       | -                | /Group/Instance/Topic  |
+| [Cyclone DDS](https://github.com/ros2/rmw_cyclonedds/pull/303/files)              | -       | Type Name        | Topic Name             |
```

Comment:
@budrus @FerdinandSpitzschnueffler 

IMHO we should unify the mapping of DDS gateway, Cyclone and rmw as they are basically all just DDS. However, I'm unsure about the consequences if we adapt the mapping e.g. in rmw. @budrus Can you elaborate those?

Reply:
@jeising I just had a look at `rmw_iceoryx` and found out that you created [the mapping](https://github.com/ros2/rmw_iceoryx/blob/foxy/rmw_iceoryx_cpp/src/internal/iceoryx_name_conversion.cpp#L102-L147). Do you have details on why you chose this mapping? I'm currently in favor to go for the mapping that Cyclone DDS choose (Instance = Type Name, Event = Topic Name).

Side question: Do you fancy a hackathon in the next weeks updating rmw_iceoryx with iceoryx v1.0? :sunglasses: There is already a [PR](https://github.com/ros2/rmw_iceoryx/pull/42) that we could finish off.

Reply:
Let's have a look into what would be a good mapping. I guess there were multiple options. It should allow to map native iceoryx msgs as well as native ros msgs. Looking forward to the hackathon!

Reply:
@mossmaurice The mapping in rmw_iceoryx is based on a mapping from Autosar Adaptive to ROS. I.e. how to convert a service/instance/event into a topic/namespace/type and vice versa. We had questions like "what will be shown as type in ros_topc list if we have a publisher that was created with ara::com API" This is different from what we do in Cyclone where we just do a mapping of DDS topic and type to the three CaPro IDs without any consideration of an AUTOSAR world. If you just adapt rmw to the another simple mapping these mixing of worlds is no more considered. I think @jeising wanted to have the possibility to debug ara based applications with ROS tools